django/core/exceptions.py

```diff
@@ -71,6 +71,11 @@ class ImproperlyConfigured(Exception):
     pass
 
 
+class FilePermissionError(Exception):
```

Comment:
I'm not sure the description of `PermissionError` really makes sense here.  However, I do like the idea of subclassing `OSError`.